app/src/main/java/com/geeksville/mesh/model/Node.kt

```diff
@@ -121,13 +121,26 @@ data class Node(
             null
         }
         val humidity = if (relativeHumidity != 0f) "%.0f%%".format(relativeHumidity) else null
+        val soilTemp = if (soilTemperature != 0f) {
+            if (isFahrenheit) {
```

Comment:
Do we really not have a real to yankee converter function? 
Given how many places it is used, it really should be one utility, not written in each spot. 

But it this is how it is in the other places, I'll allow it. Especially as they apparently aren't working at the moment anyway.